CI.md

```diff
@@ -0,0 +1,32 @@
+# Commands to trigger ci pipeline
+
+## Guide
+
+1. ci pipeline will be triggered when your comment on pull request matched command.
+2. Some ci pipelines only response to people belong to specific organization like pingcap & tikv.
```

Comment:
I think we're in pingcap/tidb so you can just talk about this repo itself.

BTW, what about consistently use "CI" in content and rename filename to "ci.md"?